### PR TITLE
[1102] Fix the list item label positioning

### PR DESCRIPTION
**Pull request by lfasani**
Add the 'label:inside-begin' label type.
Update the front and the ELK layout

Bug: https://github.com/eclipse-sirius/sirius-components/issues/1102
Signed-off-by: Laurent Fasani <laurent.fasani@obeo.fr>

### Type of this PR 

- [X] Bug fix

### How to test this PR?

- [ ] Frontend Unit tests
- [ ] Backend Unit tests
- [ ] Cypress : please specify
- [X] Manual Test :
-Diagram Edition - Support List containers

### CODE REVIEW

**Review by sbegaudeau — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/422587/158556222-759335b1-09fd-4db1-944b-4649d517ab4f.mov

It's better for the incremental layout but worse for the ELK layout... Have you tested your solution?

File: backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/export/svg/ImageRegistry.java [hunk 1]
```diff
@@ -144,11 +144,6 @@ private String updateSvgContent(byte[] content) {
 
         String cleanSvgString = new String(content).replaceAll(xmlDeclaration, ""); //$NON-NLS-1$
 
-        String svgWidthPattern = "width=\".*?\""; //$NON-NLS-1$
-        cleanSvgString = cleanSvgString.replaceFirst(svgWidthPattern, ""); //$NON-NLS-1$
-        String svgHeightPattern = "height=\".*?\""; //$NON-NLS-1$
-        cleanSvgString = cleanSvgString.replaceFirst(svgHeightPattern, ""); //$NON-NLS-1$
-
```
Comment (lfasani) [inline backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/export/svg/ImageRegistry.java:147]: This part has fixed https://github.com/eclipse-sirius/sirius-components/issues/1106 in addtion to the label icon display
I am sorry I can not remember the use case where this code was useful

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -57,6 +57,7 @@
 - [diagram] Fix an issue preventing the resizing of a node if the cursor had not moved after a previous resizing
 - https://github.com/eclipse-sirius/sirius-components/issues/1075[#1075] [compatibility] Use the proper icon feature for OperationAction
 - https://github.com/eclipse-sirius/sirius-components/issues/1104[#1104] [diagram] Fix the image base path during the SVG export
+- https://github.com/eclipse-sirius/sirius-components/issues/1104[#1102] [diagram] Fix the node label positioning (rectangle, list and list item)
```
Comment (sbegaudeau) [inline CHANGELOG.adoc:60]: The URL is not valid, which is why the build is failing. Please keep an eye on the results of the build of your pull requests

File: backend/sirius-components-starter/src/main/java/org/eclipse/sirius/components/starter/configurationproperties/RefererConfigurationProperty.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    private boolean enabled;
+
+    public RefererConfigurationProperty(boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    public boolean getEnabled() {
+        return this.enabled;
+    }
```
Comment (sbegaudeau) [inline backend/sirius-components-starter/src/main/java/org/eclipse/sirius/components/starter/configurationproperties/RefererConfigurationProperty.java:30]: No need for two getter for the same variable

**Review by sbegaudeau — COMMENTED** [review]
I'll update the PR with the minor fixed required

**Review by sbegaudeau — APPROVED** [review]

**Pull request merged**